app/controllers/admin_shelters_controller.rb

```diff
@@ -0,0 +1,8 @@
+class AdminSheltersController < ApplicationController
```

Comment:
My reading of the user story and what I remember from some classes where we talked about the project is that we'd created a shelters_controller.rb within an Admin sub-directory of our controllers folder, rather than a controller called "AdminShelters".  

From the intro to the Approving Applications User Stories: "For this set of stories, we will be making routes that begin with '/admin'. This is to indicate that only a user with special privileges should be able to accept or reject pets on an application. Normally, we would want to make sure that a user is logged into an admin account before being able complete any of this workflow, but we will not add any log in or authorization functionality to this project."